### ISSUE TITLE
The plugin 'Markdown' can not be loaded

### ISSUE THREAD

**Issue by TPei** [GeorgesStavracas/gedit-markdown#6]
Hey, when I try to enable the plugin in gedit -> preferences -> plugins -> try to place checkmark, I get the error message "The plugin 'Markdown' can not be loaded".
Any idea if this still works? I run gedit 3.20.2. Thanks :)

**Comment by drxspace** [comment]
Same situation here too `gedit 3.22.0`

**Comment by sixtyfive** [comment]
+1, also 3.22.0. It's a shame because it just seems to be something small that went out of date.

**Comment by manfreed** [comment]
A possible solution is to install the package called python3-markdown. Note that the readme says to install python-markdown, which is for python 2.7.

**Comment by sixtyfive** [comment]
@manfreed I take it you're referring to Debian-based distributions?

**Comment by manfreed** [comment]
Yes, specifically to Ubuntu 17.10, but the same might apply to other Debian
based distros too.

**Comment by slottwo** [comment]
Same problem here.
OS: Pop! 21.04
Gedit: 3.38.1
Gnome Shell: 3.38.4
python3-markdown: 3.3.4-1
python-markdown has no candidate for installation
